python/tests/plugins/test_gen_protobuf.py

```diff
@@ -73,9 +74,9 @@ def test_gen_protobuf(self):
                 self.assertIn('Description for the DataA MessageMetadataData', data_a_proto_file_contents)
 
                 # Assert data_a structure
-                self.assertIn("MessageMetadataData metadata", data_a_proto_file_contents)
+                self.assertIn("messageMetadataData metadata", data_a_proto_file_contents)
```

Comment:
The original value is the desired output.

This is also the case for the other edited lines between here and line 118.